CommunicationServer/Domain/DTOs/ResponseJoinedGameDto.cs

```diff
@@ -0,0 +1,9 @@
+namespace Domain.DTOs;
+
+public class ResponseJoinedGameDto
```

Comment:
Consider a different name that says its a stream and not a response

Reply:
Fixed, changed it to JoinedGameStreamDto